.github/workflows/quay.yml

```diff
@@ -29,6 +29,7 @@ on:
     branches:
       - master
       - stage_*
```

Comment:
Please comment, I'm not sure I understand correctly.

Reply:
removed. Was a test of GitHub actions